spec/rails_helper.rb

```diff
@@ -3,6 +3,7 @@
 require File.expand_path('../../config/environment', __FILE__)
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.